Hush/Assets/Scripts/Player/PlayerPossessions.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+public class PlayerPossessions : MonoBehaviour
```

Comment:
This class will probably be destroyed depending on how siobhan implements things

Reply:
PlayerInventory data could be stored in the GameState object i mentionned earlier and all this script would do is update the GameState. That way doors, enemies, whatever else we want would be able to know if the player has the keys.

Reply:
GameMaster is in with PR #47 😄

Reply:
done in 0bf2612 (integrated with what Siobhan did)